utils/__tests__/dashboard-helper.test.js

```diff
@@ -105,14 +116,15 @@ describe('dashboard-helper', () => {
       });
 
       const output = await runHelper('http://localhost', 'token');
+      console.debug('output: ', output)
```

Comment:
Do we still want this console.debug here? or do think its necessary to understand the test